multi_agents/agents/editor.py

```diff
@@ -42,7 +40,6 @@ async def plan_research(self, research_state: dict):
             "role": "user",
             "content": f"Today's date is {datetime.now().strftime('%d/%m/%Y')}\n."
                        f"Research summary report: '{initial_research}'\n"
-                       f"{f'Human feedback: {human_feedback}. You must plan the sections based on the human feedback.' if include_human_feedback else ''}\n"
```

Comment:
@ElishaKay why did you remove it? It's critical for updating the plan based on the human feedback

Reply:
There were two edge cases I needed to handle (when human_feedback = no or human_feedback = blank), therefore, I needed to refactor a bit

In other words, instead of processing the human_feedback in `editor_agent.plan_research` I processed it one function down in `editor_agent.run_parallel_research`

 Here's the gist of the new processing logic:
 
 ```
 human_feedback = research_state.get("human_feedback")

final_drafts = [
    chain.ainvoke({
        "task": research_state.get("task"),
        "topic": query + (f". Also: {human_feedback}" if human_feedback is not None else ""),
        "title": title,
        "headers": self.headers
    })
    for query in queries
]
```

It's a pretty solid approach, in my opinion, since the GPTR class will then factor in the human_feedback when generating the subqueries in the remainder of the flow